src/table/types.ts

```diff
@@ -78,6 +78,15 @@ export interface ContextValue {
   selectionDispatch: SelectionDispatch;
   hoverIndex: number;
   setHoverIndex: React.Dispatch<React.SetStateAction<number>>;
+  baseProps: {
+    selectionsAPI: ExtendedSelectionAPI;
+    layout: TableLayout;
+    model?: EngineAPI.IGenericObject;
```

Comment:
I think we should make sure model is defined in the index file

Reply:
I had to make it optional as the paginiation table does not pass the model

Reply:
ah I see, then we shouldn't change anything in the index 🙃